pkg/cmd/uninstall/hubaddon/exec.go

```diff
@@ -0,0 +1,121 @@
+// Copyright Contributors to the Open Cluster Management project
+package hubaddon
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	addonclientset "open-cluster-management.io/api/client/addon/clientset/versioned"
+	"open-cluster-management.io/clusteradm/pkg/helpers/reader"
+
+	"github.com/spf13/cobra"
+	"k8s.io/klog/v2"
+
+	"open-cluster-management.io/clusteradm/pkg/cmd/install/hubaddon/scenario"
+)
+
+func (o *Options) complete(cmd *cobra.Command, args []string) (err error) {
+	klog.V(1).InfoS("addon options:", "dry-run", o.ClusteradmFlags.DryRun, "names", o.names)
+	return nil
+}
+
+func (o *Options) validate() (err error) {
+	err = o.ClusteradmFlags.ValidateHub()
+	if err != nil {
+		return err
+	}
+
+	if o.names == "" {
+		return fmt.Errorf("names is missing")
+	}
+
+	names := strings.Split(o.names, ",")
+	for _, n := range names {
+		switch n {
+		case scenario.AppMgrAddonName:
+			continue
+		case scenario.PolicyFrameworkAddonName:
+			continue
+		default:
+			return fmt.Errorf("invalid add-on name %s", n)
+		}
```

Comment:
Similar to the deployment, I was thinking this could also be something like:
```golang
		if _, ok := scenario.AddonDeploymentFiles[n]; !ok {
			return fmt.Errorf("invalid add-on name %s", n)
		}
```